src/storhaug

```diff
@@ -399,6 +402,7 @@ setup_create_resources()
             config="${HA_NFS_CONF}" \
             state_fs="${HA_NFS_TYPE}" \
             state_mnt="${HA_MNT_DIR}/${HA_NFS_MNT_DIR}" \
+        op monitor on-fail="block" \
```

Comment:
Yes, it maybe configurable how many times a resource shall be restarted, but it is still good to restart the broken one only after the problem is solved(by administrator, manually) and the resource can go on to serve a good job stably.